### ISSUE TITLE
Validation reports errors on lists

### ISSUE THREAD

**Issue by andrew-garner** [hashicorp/vscode-terraform#183]
**Environment**
vscode 1.35.0
vscode-terraform 1.3.11
terraform v0.12.1

In VSCode I get problems reported from terraform files when using sub-elements from numbered items from list. Running `terraform validate` reports no errors with this syntax, and `terraform apply` runs successfully, it's just in VSCode with this extension that this line is underlined in red, the file and project is highlighted in red in the file explorer and all occurrences of this syntax are listed as problems. 

internal_ip in the snippet below will be highlighted as an error `expected "}" but found "."`

```
resource "vcd_dnat" "ssh" {
  org             = "${var.vcd_org}"
  vdc             = "${var.vcd_vdc}"
  edge_gateway    = "${var.vcd_edgegateway}"
  external_ip     = "${var.public_ip}"
  port            = 22022
  internal_ip     = "${vcd_vapp_vm.bastion.network[0].ip}"
  translated_port = 22
}
```

**Comment by mauve** [comment]
this is probably because we do not support terraform 0.12 yet (see #157), you can try to disabled `indexing` in the settings `terraform.indexing.enabled = false`

**Comment by paultyng** [comment]
We just released [v2.0.0-rc.1](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) of the extension. The main features include:

* Added syntax support for 0.12
* Added [terraform-ls](https://github.com/hashicorp/terraform-ls) usage by default (currently on 0.3.0, which offers basic provider code completion)

You can find additional information and specifics in the [release notes](https://github.com/hashicorp/vscode-terraform/releases/tag/v2.0.0-rc.1) and [CHANGELOG](https://github.com/hashicorp/vscode-terraform/blob/master/CHANGELOG.md).

With this release we expect that many of the prior issues and PRs are no longer relevant or have been addressed, and are therefore being closed. If you feel the action taken on an issue or PR is in error, please comment as such and we can figure out the appropriate way to address it.

We plan to add the final 2.0.0 release to the marketplace soon, but are actively seeking your feedback now on the release candidates. You can download the .vsix from the releases page and [manually install it in VS Code](https://code.visualstudio.com/docs/editor/extension-gallery#_install-from-a-vsix) to try it out.
